frontend/packages/console-plugin-sdk/src/registry.ts

```diff
@@ -55,9 +56,19 @@ export class ExtensionRegistry {
       );
   }
 
+  public getBlockerFlags(typeGuards: ExtensionTypeGuard<ExtensionWithFlags>[]) {
```

Comment:
```suggestion
  public getDisallowedFlags(typeGuards: ExtensionTypeGuard<ExtensionWithFlags>[]) {
```

Reply:
FYI, once #3383 is merged, we can start converting code that consumes extensions, from using `plugins.registry` to using `useExtensions` hook or `withExtensions` HOC. The new mechanism automatically responds to changes in `RootState.FLAGS` and honors the `flags` object which is part of `Extension` type itself.